2024-frontend/src/components/ImportStaffCallout.vue

```diff
@@ -1,39 +1,16 @@
 <script setup>
-import AppCode from "@/components/AppCode.vue"
-
 const file = "/static/documents/admin_importer_des_cantines_exemple_ma_cantine.csv"
 </script>
 
 <template>
   <DsfrCallout title="En tant que membre de l'équipe ma cantine" titleTag="h3">
     <div class="fr-grid-row fr-grid-row--bottom fr-mt-3w">
       <div class="fr-col-12 fr-col-md-8">
-        <p>
-          Vous pouvez ajouter 3 colonnes additionnelles à la fin du fichier CSV :
+        <p class="fr-mb-1w">
+          Vous pouvez ajouter 3 colonnes additionnelles à la fin du fichier CSV : admin_ministère_tutelle,
```

Comment:
```suggestion
          Pour utiliser l'import admin, vous devez ajouter 3 colonnes additionnelles à la fin du fichier CSV : admin_ministère_tutelle,
```

Est-ce qu'on montre que c'est obligatoire ?

Reply:
yeah !